### PR TITLE
dataflow: use us-central1-f for gpu test

### PR DESCRIPTION
**Pull request by davidcavazos**
## Description

It won't fix the timeouts, but it should fix the quota issues.

Fixes #5275, #5395, #5397

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by leahecole — APPROVED** [review]
LGTM if tests pass :)

**Review by busunkim96 — APPROVED** [review]

**Comment by davidcavazos** [comment]
The workaround was more around this [comment](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/5067/files#r562166080). I missed that comment before merging the PR, and for some reason I thought I already made the changes in another PR, but apparently not.

**Pull request merged**
